### PR TITLE
Add black-stm32f4xx board

### PR DESCRIPTION
**Pull request by eugene-bright**
More info on this board:
https://os.mbed.com/users/hudakz/code/STM32F407VET6_Hello/

### CODE REVIEW

File: boards.stm32.mk [hunk 1]
```diff
@@ -116,6 +116,7 @@ $(eval $(call stm32f4board,nucleo-f410rb,GPIOA,GPIO5))
 $(eval $(call stm32f4board,nucleo-f446re,GPIOA,GPIO5))
 $(eval $(call stm32f4board,olimex-p405,GPIOC,GPIO12))
 $(eval $(call stm32f4board,olimex-e407,GPIOC,GPIO13))
+$(eval $(call stm32f4board,black-stm32f4xx,GPIOA,GPIO6))
```
Comment (karlp) [inline boards.stm32.mk:119]: can we call it the black stm32f407ve board at least?  https://forum.micropython.org/viewtopic.php?t=3086 mentions that the 407ve and the 407ze boards have the led on different pins, (PA6/7 vs PF9/10 for led1/2 respectively)

Comment (karlp) [reply boards.stm32.mk:119]: https://jeelabs.org/2018/retrocomputing-on-f407/ shows a 407vg board with the led on PE0 too, so "f4xx" is not really a good enough id for your board I'm afraid :)

Comment (eugene-bright) [reply boards.stm32.mk:119]: You are right. My board is labeled as `STM32_F4VE v2.0` build `1509`.
Should I add version number in addition to the chip name?

Comment (karlp) [reply boards.stm32.mk:119]: version seems useful, can probably skip the build 1509...

**Comment by eugene-bright** [comment]
I will clarify branch name and open new request.

**Pull request closed without merging**